examples/EnergyMeterPulseSensor/EnergyMeterPulseSensor.ino

```diff
@@ -46,10 +46,10 @@
 #include <MySensors.h>
 
 #define DIGITAL_INPUT_SENSOR 3  // The digital input you attached your light sensor.  (Only 2 and 3 generates interrupt!)
-#define PULSE_FACTOR 1000       // Number of blinks per of your meter
```

Comment:
Maybe change to kWh? (Lowercase k) 
https://en.wikipedia.org/wiki/Kilo-

Reply:
Turns out I can't edit the code anyway since there is a complicated privacy-unfriendly proces for singing over copyright

Feel free to make the edit yourself?